.github/CHANGELOG.md

```diff
@@ -2,6 +2,55 @@
 
 <h3>New features since last release</h3>
 
+* The transform for the Jacobian of the classical preprocessing within a QNode,
+  `qml.transforms.classical_jacobian`, now takes a keyword argument `argnums` to specify
+  the QNode argument indices w.r.t which the Jacobian is computed.
+  [(#1645)](https://github.com/PennyLaneAI/pennylane/pull/1645)
+
+  The output behaviour with `argnums=None` differs slightly for different interfaces,
+  following the respective output shape of the native `jacobian` equivalent:
+
+  | Interface    | `argnums=None`     | `type(argnums)=int`   | `argnums=Sequence[int]`   |
+  | ------------ | ------------------ | --------------------- | ------------------------- |
+  | `'autograd'` | `tuple(arrays)`\*  | `array`               | `tuple(array)`            |
+  | `'jax'`      | `array`            | `array`               | `tuple(array)`            |
+  | `'tf'`       | `tuple(arrays)`    | `array`               | `tuple(array)`            |
+  | `'torch'`    | `tuple(arrays)`    | `array`               | `tuple(array)`            |
+
+  \* If all QNode argument are scalars, the tuple is unpacked and the one-dimensional Jacobian
+     arrays are stacked into one `array`. This is due to the behaviour of `qml.jacobian`.
+
+  An example for the usage of `argnums` is
+
+  ```python
+  @qml.qnode(dev)
+  def circuit(x, y, z):
+      qml.RX(qml.math.sin(x), wires=0)
+      qml.CNOT(wires=[0, 1])
+      qml.RY(y ** 2, wires=1)
+      qml.RZ(1 / z, wires=1)
+      return qml.expval(qml.PauliZ(0))
+
+  jac_fn = qml.transforms.classical_jacobian(circuit, argnums=[1, 2])
+  ```
+
+  The Jacobian can then be computed at specified parameters.
+  Note that unlike the behaviour in `tensorflow`, _all_ QNode arguments have to be provided
```

Comment:
```suggestion
  Note that unlike the behaviour in `tensorflow`, *all* QNode arguments have to be provided
```

(just in case, I recall there were some issues using \_ when converting from markdown -> rest in the past)

Reply:
Thanks. Yes, I noticed that I started mixing rest and markdown syntax quite a bit in my head :sweat_smile: